include/inviwo/core/rendering/datavisualizermanager.h

```diff
@@ -72,9 +72,11 @@ class IVW_CORE_API DataVisualizerManager {
     std::vector<FileExtension> getSupportedFileExtensions() const;
 
     /**
-     * Get a list of Data Visualizers supporting the supplied extension.
+     * \brief Get a list of Data Visualizers supporting the file extension of the supplied string.
+     * @param filePathOrExtension Path to file, or simply the extension.
      */
-    std::vector<DataVisualizer*> getDataVisualizersForExtension(const std::string& ext) const;
+    std::vector<DataVisualizer*> getDataVisualizersForExtension(
```

Comment:
I think that it is only used in one or two places. I changed started changing it at one point to exactly your suggestion, but thought that it might be good to keep backwardscompatibility and consistency with the DataReader/DataWriter